doc/user_guide/lfric.rst

```diff
@@ -1961,15 +1961,27 @@ operates_on                    Data passed for each field/operator argument
 ``cell_column``                Single column of cells from the owned region (except
                                when performing an INC operation on continuous fields
 			       when it will include one level of halo cells).
+``owned_cell_column``          As ``cell_column`` but the iteration space must *only*
+                               cover owned columns (i.e. excluding halo columns).
 ``halo_cell_column``           Single column of cells exclusively from halo region.
 ``owned_and_halo_cell_column`` Single column of cells but iteration space will include
                                both owned and halo regions.
-``dof``                        Single DoF .
+``dof``                        Single DoF.
+``owned_dof``                  Single DoF but the iteration space must *only* include
+                               owned DoFs (i.e. not annexed or halo DoFs).
 ``domain``                     All columns of cells in the (sub-)domain.
 ============================== =======================================================
 
-(For a description of the concepts of 'owned' and 'halo' cells please see the
-:ref:`lfric-developers`.)
+(For a description of the concepts of 'owned' and 'halo' cells and 'annexed' DoFs
+please see the :ref:`lfric-developers`.)
```

Comment:
Aha, here are the pictures I was looking for. This may be a separate PR, but I think this needs reorganising.

I already think that this operates_on section is too deep in this page, some preceding text already assumes I know about these. But the LFRic concepts explained inside "Developer Guide > Generic Code" should definitely be introduced earlier in this page.

Unfortunately it is not as simple as copy paste because that section also eventually goes too deep into internal details, but the basic concepts at the top would be useful.